yt_dlp/extractor/openrec.py

```diff
@@ -21,7 +21,7 @@ def _expand_media(self, video_id, media):
             if not m3u8_url:
                 continue
             yield from self._extract_m3u8_formats(
-                m3u8_url, video_id, ext='mp4', m3u8_id=name)
+                m3u8_url, video_id, ext='mp4', m3u8_id=name, headers={'Referer': 'https://www.openrec.tv/'})
```

Comment:
could be a baseclass var, e.g.
```py
    _M3U8_HEADERS = {'Referer': 'https://www.openrec.tv/'}
```
```suggestion
                m3u8_url, video_id, ext='mp4', m3u8_id=name, headers=self._M3U8_HEADERS)
```
and same for the other lines where it's used